src/main/java/hudson/tasks/Ant.java

```diff
@@ -341,7 +354,8 @@ public void buildEnvVars(EnvVars env) {
          * Gets the executable path of this Ant on the given target system.
          */
         public String getExecutable(Launcher launcher) throws IOException, InterruptedException {
-            return launcher.getChannel().call(new Callable<String,IOException>() {
+            return launcher.getChannel().call(new MasterToSlaveCallable<String,IOException>() {
+                private static final long serialVersionUID = 906341330603832653L;
```

Comment:
Probably a stupid question but, why is it safer to have it in a `static` nested class? Also, the callable calls `getExeFile` which is an instance method I could not access from a `static` nested class. Thanks.